sriracha/helper/manage_user_groups.py

```diff
@@ -0,0 +1,98 @@
+import crypt
+import grp
+import pwd
+import spwd
+import subprocess
+import sys
+
+def is_user_exists(username):
+    """Checks User existence"""
+    try:
+        pwd.getpwnam(username)
+        return True
+    except:  # pylint: disable=bare-except
+        log.info("User someusr does not exist.")
+
+    return False
+
+
+def is_group_exists(groupname):
+    """Checks group existence"""
+    try:
+        grp.getgrnam(groupname)
+        return True
+    except:  # pylint: disable=bare-except
+        log.info("Group %s does not exist.", groupname)
+
+    return False
+
+
+def check_passwd(username, expected_passwd):
+    """Checks user passwd"""
+    try:
+        password = spwd.getspnam(username).sp_pwdp
+        return password == crypt.crypt(expected_passwd, password)
+    except:  # pylint: disable=bare-except
+        log.info("Failing to get passsword for the user %s", username)
+
+    return False
+
+
+def check_gid(groupname, expected_gid):
```

Comment:
This comment and function name is ambiguous, what does it do?

Reply:
It checks the expected `group_id` is set for the `group`.